alf/algorithms/sac_algorithm.py

```diff
@@ -182,6 +184,10 @@ def __init__(self,
             q_network (Callable): is used to construct QNetwork for estimating ``Q(s,a)``
                 given that the action is discrete. Its output spec must be consistent with
                 the discrete action in ``action_spec``.
+            reward_weights (list[float]): this is only used when the reward is
+                multidimensional. In that case, the weighted sum of the q values
+                is used for training the actor.
```

Comment:
In that case, the weighted sum of the q values is used for training the actor if reward_weights is not None.
Otherwise, the sum of the q values is used.